Lombiq.Gulp.Extensions.csproj

```diff
@@ -26,6 +26,7 @@
   -->
   <Target Name="InitCommonNodeModules" BeforeTargets="ResolveReferences" Inputs="package.json" Outputs="$(InitCommonNodeModulesStampFile)">
     <Exec Command="npm install" />
+    <RemoveDir Directories="../../node_modules" Condition="Exists('node_modules') and Exists('../../node_modules')" />
```

Comment:
@Piedone ilyennel mi nem próbálkoztunk mi már? Valami rémlik, és nem volt ilyen egyszerű.

Reply:
Erről nincs konkrét emlékem, de nézd meg a file history-ját. Alapból nem látok benne problémát, de az értelmét sem látom igazán: mi szükség erre a lépésre? Eddig is működött, szóval gondolom a `move` önmagában is OK, akkor is, ha már létezik a mappa.

Reply:
Tudom miért csinálta Dávid, van, hogy beragad valami branch váltáskor, és próbálná másolni a node_modules mappát, de nem tudja, mert ott van a mappa és elfailel, amit manuálisan kell megjavítani (törölni a mappát). Szal én lehet egyelőre ezt bennehagynám. BTW valszeg ez most csak azért okozott problémát @DAud-IcI , mert ezen a branchen még nem volt ott a Lukó kódja az ES Lint-tel, ami különösen bekavart valamit, mikor oda vissza váltogattál. Normál esetben branch váltások között nincs erre szükség.

Reply:
De akkor ennek mindig elő kellene jönnie. Miért és pontosan mikor ragad be? Mert én még egyszer sem tapasztaltam. Vagy ez lenne néha az access denied-ok mögött (bár akkor nem értem a remove miért működne)?

Reply:
Ha két branchben packages.json nem egyezik, akkor váltáskor mindig előjön.

Reply:
Jó, de mi jön elő, mi a hibajelenség?

Reply:
Azt mondja move node_modules ../.. returned 1 és ezzel elbukik a build. Utána letörlöm az /src/node_modules mappát és újra buildelek, probléma megoldódik, de nagyon random.

Reply:
Aha. Hát akkor pont ilyet nem láttam. OK akkor.